webdriver-spec.html

```diff
@@ -2410,7 +2415,8 @@
 }</pre>
 </aside>
 
-<p>The <a>remote end steps</a> are:
+<p>The <a>remote end steps</a> <a data-lt="RES-arguments">with arguments</a> <var>url variables</var>
+ and <var>parameters</var> are:
```

Comment:
Thanks for the input. I'll do some more and see how the text looks and I'll continue with the rest of the comments/changes.